opm/input/eclipse/Schedule/Group/GuideRate.cpp

```diff
@@ -277,7 +277,7 @@ void Opm::GuideRate::group_compute(const std::string& wgname,
         }
 
         if (is_formula) {
-            const auto guide_rate = this->eval_form(config.model(), oil_pot, gas_pot, wat_pot);
+            const auto guide_rate = this->eval_form(config.model(), wgname, oil_pot, gas_pot, wat_pot);
```

Comment:
If you pass `"group " + wgname` (and `"well " + wgname` at the other call site) as an argument here instead of "just" `wgname`, then you don't need the somewhat clunky `well/group` construction in `eval_form()`.